lib/Alchemy/Phrasea/SearchEngine/Elastic/Indexer/RecordIndex.php

```diff
@@ -73,6 +73,8 @@ public function getMapping()
         $mapping->addIntegerField('height')->disableIndexing();
         $mapping->addIntegerField('size')->disableIndexing();
 
+        $mapping->addGeoPointField('location')->disableIndexing();
```

Comment:
do NOT disableIndexing, else kibana :( 
ALREADY fixed in master afert this pr was merged.